### DISCUSSION TITLE
useBalance will work even enabled option is false

### DISCUSSION THREAD

**Discussion by IvanZen** [wevm/wagmi#1903] [Q&A]
This can be reproducted with code below.
`
const { address } = useAccount();

const { data: balance } = useBalance({
address,
enabled: false,
});
`
Is this an expected behavior?

**Comment by jxom** [comment]
The Hook will not fetch, but if you have a cached value, it will display that. Might be best to use local React state if you wish to hide the cached value of `useBalance`.
